wurst/systems/spells/JumpSystem.wurst

```diff
@@ -0,0 +1,112 @@
+package JumpSystem
+import ChatCommands
+import GameConfig
+import ClosureForGroups
+import ClosureTimers
+
+constant let DEFAULT_GRAVITY = 175.
+constant let STORMCROWFORM_ID = 'Amrf'
+
+init
+    registerCommandAll("jump") (triggerPlayer, args) ->
+        if (gameConfig.isTestModeEnabled() == true)
+            forUnitsSelected(triggerPlayer) u ->
+                if (u.isType(UNIT_TYPE_HERO))
+                    waterFallJump(u, u.getPos().polarOffset(angle(0), 400))
+        else
+            printTimedToPlayer("Requires test mode!", 5., triggerPlayer)
+        
+public function waterFallJump(unit u, vec2 targetPos)
```

Comment:
I'd not reference `waterFall` (which is one word by the way) here if this is a generic jump function.

Reply:
Its not a generic function, since it plays animations that require certain troll models with the backflip stand animation. Every different type of jump should have their own function in my opinion.

Reply:
Also renamed this to ThiefJump now that its used for both waterfalls and jump ability